docs/learn/dapp-staking.md

```diff
@@ -0,0 +1,319 @@
+---
+sidebar_position: 11
+title: dApp Staking
+toc_max_heading_level: 3
+---
+
+## Introduction
+
+Astar and Shiden networks provide a unique way for developers to earn rewards by developing products that native token holders can decide to support.
+
+The principle is simple - stakers _lock_ their tokens to _stake_ on a dApp, and if the dApp attracts enough support, it is rewarded in native currency, derived from the inflation.
+In turn stakers are rewarded for locking & staking their tokens.
+
+The following chapters will provide an overview of the functionality and terminology with accompanying examples.
+
+## Functionality Overview
+
+### Eras
+
+Eras are the basic _time unit_ in dApp staking and their length is measured in the number of blocks.
+
+Eras are not expected to last long, e.g. current production networks era length is roughly 1 day (7200 blocks).
+After an era ends, it's usually possible to claim rewards for it, if user or dApp are eligible.
+
+### Periods
+
+Periods are another _time unit_ in dApp staking. They are expected to be more lengthy than eras.
+Each period is denoted by a number, e.g. **1**, which increments each time a new period begins.
+
+Each period consists of two subperiods:
+* `Voting`
+* `Build&Earn`
+
+Period beginning is marked by the `Voting` subperiod, after which follows the `Build&Earn` subperiod.
+
+Stakes are **only** valid throughout a period. When new period starts, all stakes are reset to **zero**. This helps prevent projects remaining staked due to inertia of stakers, and makes for a more dynamic staking system. Staker doesn't need to do anything for this to happen, it is automatic.
```

Comment:
This helps prevent projects remaining staked due to inertia of stakers, and makes for a more dynamic staking system.
-->
Protocol dynamic prevents inactive projects having high stakes due to inertia of stakers while at the same time allows more visibility for dApps joining the protocol and equal chance in attracting stakers' attention.

Reply:
Applied.